### PR TITLE
LPS-125255 - Updates @clayui/* dependencies

### PR DESCRIPTION
**Pull request by bryceosterhaus**
Packages Updated
 - @clayui/autocomplete@3.2.3
 - @clayui/breadcrumb@3.3.3
 - @clayui/card@3.6.3
 - @clayui/color-picker@3.4.3
 - @clayui/css@3.22.1
 - @clayui/date-picker@3.6.3
 - @clayui/drop-down@3.8.3
 - @clayui/form@3.14.1
 - @clayui/list@3.4.4
 - @clayui/localized-input@3.2.3
 - @clayui/multi-select@3.9.1
 - @clayui/multi-step-nav@3.3.3
 - @clayui/pagination-bar@3.2.3
 - @clayui/pagination@3.3.3
 - @clayui/time-picker@3.2.3
 - @clayui/toolbar@3.0.3
 - @clayui/upper-toolbar@3.1.11

Mostly consists of css changes as well as providing `className` prop for ClayToggle Component.

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
@liferay-continuous-integration u up?

**Comment by pyoo47** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

File: modules/yarn.lock [hunk 1]
```diff
@@ -1411,115 +1288,6 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
-"@emotion/cache@^10.0.27":
```
Comment (jbalsas) [inline modules/yarn.lock:1414]: Huh, similar changes to those in https://github.com/liferay-frontend/liferay-portal/pull/660... I wonder how all these modules crept in and why are they leaving now... 🤔

Comment (bryceosterhaus) [reply modules/yarn.lock:1414]: I wonder if it has to do with using the `install-scripts.sh` from the frontend-projects repo. I did that earlier today and wonder if it created any stale items in the yarn cache.

Comment (bryceosterhaus) [reply modules/yarn.lock:1414]: deduped yarn.lock and looks a little better now

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

File: modules/yarn.lock (hunk 1 above)
Comment (bryceosterhaus) [reply modules/yarn.lock:1414]: nvm, still looks wild

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
@john-co could you take a look at these? I don't think any of the clay changes we added would cause these failures

**Comment by john-co** [comment]
Reviewed. Test failures are unrelated. :heavy_check_mark: 

- same reasons as https://github.com/liferay-frontend/liferay-portal/pull/618#issuecomment-754927319
- search integration test is flaky: https://issues.liferay.com/browse/LPS-123052

**Comment by bryceosterhaus** [comment]
forwarded at https://github.com/brianchandotcom/liferay-portal/pull/97439

**Pull request closed without merging**